docs/usermanual.md

```diff
@@ -193,17 +193,18 @@ Some additional remarks regarding polling commands:
 MQTT is a machine-to-machine (M2M)/IoT connectivity protocol, focused on a lightweight interaction between peers. MQTT
```

Comment:
I'd suggest to add the following entry to CHANGEX_NEXT_RELEASE:

```
- Add: /api-docs endpoint provising swagger-based documentation
```

or something similar

Reply:
fixed in 2808c0b11be011d26b2a3c9e0077e1c4a22c056c